imports/ui/pages/components/Popup/Popup.js

```diff
@@ -48,33 +44,26 @@ class GameRequestPopup extends Component {
             {title}
           </h3>
           <button
-            onClick={this.gameRequestHandler.bind(this, "gameAccept", requestId)}
-            style={this.props.cssManager.innerPopupMain()}
+            onClick={() => this.gameRequestHandler("gameAccept", requestId)}
+            style={cssManager.innerPopupMain()}
           >
-            {translator("accept")}
+            {translate("accept")}
           </button>
           <button
-            onClick={this.gameRequestHandler.bind(this, "gameDecline", requestId)}
-            style={this.props.cssManager.innerPopupMain()}
+            onClick={() => this.gameRequestHandler("gameDecline", requestId)}
+            style={cssManager.innerPopupMain()}
           >
-            {translator("decline")}
+            {translate("decline")}
           </button>
         </div>
       </div>
     );
   }
 }
 
+const EnhancedGameRequestPopup = translate("Common.GameRequestPopup")(GameRequestPopup);
```

Comment:
Move each component from this file to separate file. Rename this file to `index.js`. This file should import all ell popup components and export them so you can access each single popup components via ``import { PopupComponent } from ''Popup";`

File structure example:
```
/ Popup
  - index.js
  - PopupComponent1.jsx
  - PopupComponent2.jsx
  ...
```